examples/visualisations/margins.py

```diff
@@ -0,0 +1,146 @@
+import logging
+from collections import namedtuple
+from math import exp
+
+from vega_sim.null_service import VegaServiceNull
+
+WalletConfig = namedtuple("WalletConfig", ["name", "passphrase"])
+
+MM_WALLET = WalletConfig("mm", "pin")
+MM_WALLET2 = WalletConfig("mm2", "pin2")
+TRADER_WALLET = WalletConfig("trader", "password123")
+
+wallets = [MM_WALLET, MM_WALLET2, TRADER_WALLET]
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
+
+    with VegaServiceNull(
+        run_with_console=True,
+        start_order_feed=True,
+        retain_log_files=True,
```

Comment:
If things are working I'd set this to False; it's not just the log files but also vega home so chain, data node... and even the log can be GB - though probably not from this script.

Reply:
Good call, thanks - got rid of these now.